django-stubs/views/decorators/gzip.pyi

```diff
@@ -0,0 +1,3 @@
+from typing import Callable
+
+def gzip_page(view_func: Callable) -> Callable: ...
```

Comment:
Please, make it preserve `Callable` like it's done here
https://github.com/mkurnikov/django-stubs/blob/194258ab8e4767cdd8c6593792c3ec6c7697947a/django-stubs/db/transaction.pyi#L34

Reply:
Updated. Note that the other `views.decorators` methods do not preserve `Callable` like on `atomic`, so that might be something worth cleaning up in the future.